q2_quality_control/_threshold_graph/_visualizer.py

```diff
@@ -27,9 +28,11 @@
     'threshold': _PER_NUM,
     'bin_size': _PER_NUM,
     'weighted': _BOOLEAN,
+    'rep_seqs': _SKIP
 }
 
 
+#checks inputs
```

Comment:
This is no longer needed now that we're using `Range` to ensure that these values are all in the right range. We removed this.